numpy/_core/src/multiarray/multiarraymodule.c

```diff
@@ -5135,18 +5126,32 @@ PyMODINIT_FUNC PyInit__multiarray_umath(void) {
         goto err;
     }
 
-#if Py_GIL_DISABLED
-    // signal this module supports running with the GIL disabled
-    PyUnstable_Module_SetGIL(m, Py_MOD_GIL_NOT_USED);
+    return 0;
+
+ err:
+    return -1;
```

Comment:
Not a big thing, but can you just remove the `err` label?  No point in using `goto err` if there is no cleanup needed.